manifests/init.pp

```diff
@@ -6,13 +6,15 @@
 #
 class drbd(
   $service_enable = true,
-  $package_name = 'drbd8-utils',
-) {
+  Variant[String, Array[String]] $package_name = $drbd::params::package_name,
+
+) inherits drbd::params {
   include ::drbd::service
 
-  package { 'drbd':
```

Comment:
The old construction did work and simplified the other code a bit. I'm leaning to keeping the old version.